### ISSUE TITLE
Error with class_exists and SPL autoloader

### ISSUE THREAD

**Issue by lordlele** [intuit/QuickBooks-V3-PHP-SDK#94]
In these files:

```
src/XSD2PHP/src/com/mikebevz/xsd2php/Bind.php
src/XSD2PHP/src/com/mikebevz/xsd2php/Php2Xml.php
src/Core/Http/Serialization/XmlObjectSerializer.php
src/XSD2PHP/src/com/mikebevz/xsd2php/Bind.php
src/XSD2PHP/src/com/mikebevz/xsd2php/Php2Xml.php
```
the code `class_exists(whatever...` must be changed with `class_exists(whatever..., false)` so all problems with SPL Autoloader and other frameworks autoloader are fixed.

Do you need a PR?

**Comment by hlu2** [comment]
@lordlele 
What issues are we trying to resolve here? Can you provide more details?

**Comment by lordlele** [comment]
@hlu2 yes, we use your SDK in our projects with Yii, Laravel, Symfony and other frameworks.
This frameworks have their own autoloader that try to load the class when class_exists is encountered, however this create a fatal error.
With class_exists(whatever, false) the autoloader of the frameworks do not try to load the class from his own path and let QB SDK to load his own classes.

Thanks.

**Comment by hlu2** [comment]
@lordlele 
Thanks, I will change it accordingly.

**Comment by AdamDougherty** [comment]
Any update on this? The issue persists for anyone using a framework w/ autoloading

**Comment by hlu2** [comment]
@lordlele Complete. Will be released at the end of the week.
